pkg/helper/validate/validate.go

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2019 The Tekton Authors.
```

Comment:
Idea here would be to use this class for common validation methods across the CLI.